src/drivers/bitbucket_cloud.js

```diff
@@ -358,7 +358,7 @@ class BitbucketCloud {
       throw new Error('Bitbucket Cloud API endpoint not found');
 
     const headers = { Authorization: `Basic ${token}` };
-    if (body.constructor !== FormData)
+    if (body && body.constructor !== FormData)
```

Comment:
The behavior before #943 wase a bit different, methinks:
```suggestion
    if (!body || body.constructor !== FormData)
```

Reply:
your right -1 to my reading comprehension